knowledge-base/kb-numericupdown-howto-introduce-event-fired-when-repeatbuttons-are-released.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Introduce Event That is Fired When RepeatButtons are Released
+description: Implement event which is thrown when RepeatButtons are released
+type: how-to
+page_title: Add event thrown when the up or down button is let go
+slug: kb-numericupdown-howto-introduce-event-fired-when-repeatbuttons-are-released
+position: 0
+tags: numericupdown, released, value, changed, repeatbutton
+ticketid: 1405485
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>RadNumericUpDown for WPF</td>
+	</tr>
+</table>
+
+## Description
+
+By default, when the user holds down the up or down RepeatButtons of the RadNumericUpDown, its __Value__ property is continuosly updated and the __ValueChanged__ event is called each time. This article will demonstrate how to receive only one notification, when the user releases the RepeatButton.
```

Comment:
The same comment as in the following pull request - https://github.com/telerik/xaml-docs/pull/1619